src/promptflow/tests/sdk_cli_test/unittests/test_exceptions.py

```diff
@@ -0,0 +1,105 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import pytest
+from promptflow._sdk._pf_client import PFClient
+from promptflow.exceptions import ErrorInfo, ErrorCategory, ErrorTarget
+from promptflow.executor import FlowValidator
+from promptflow.executor._errors import InvalidNodeReference
+
+FLOWS_DIR = "./tests/test_configs/flows/print_input_flow"
+
+
+@pytest.mark.unittest
+class TestExceptions:
+    @pytest.fixture(autouse=True)
```

Comment:
we already have `pf` fixture: https://github.com/microsoft/promptflow/blob/main/src/promptflow/tests/sdk_cli_test/conftest.py#L29-L30, so maybe you don't need to declare this here

Reply:
Okay, I modified it.